landoscript/tests/test_android_l10n_sync.py

```diff
@@ -1,6 +1,7 @@
 import pytest
-from scriptworker_client.github_client import TransportQueryError
+from yarl import URL
```

Comment:
Unrelated changes?

Reply:
Yeah. The removal is removing an unused import, but as you say, unrelated.